client/my-sites/media-library/list-item-image.jsx

```diff
@@ -80,6 +80,7 @@ module.exports = React.createClass( {
 		var url = MediaUtils.url( this.props.media, {
 			photon: this.props.thumbnailType === MEDIA_IMAGE_PHOTON,
 			maxWidth: this.props.maxImageWidth,
+			size: this.props.thumbnailType === MEDIA_IMAGE_THUMBNAIL ? 'medium' : false,
```

Comment:
It definitely makes sense - MediaUtils is pretty opaque - although maybe separately, as I'm unsure of the wider impact.